src/node/app.ts

```diff
@@ -31,6 +31,8 @@ const listen = (server: http.Server, { host, port, socket }: ListenOptions) => {
       server.off("error", reject)
       server.on("error", (err) => util.logError(logger, "http server error", err))
 
+      if (socket && permission) fs.chmod(socket, permission)
```

Comment:
We might want to catch any errors and log them; Node can be a bit weird about uncaught promise rejections:

```suggestion
      if (socket && permission) {
        fs.chmod(socket, permission).catch((error) => {
          util.logError(logger, "socket chmod", error)
        })
      }
```

Although maybe should this be a hard exit.  If someone is setting permissions and those fail to set maybe we should be really noisy about it.  For example we could do:

```suggestion
      if (socket && permission) {
        fs.chmod(socket, permission).then(resolve).catch(reject)
      } else {
        resolve()
      }
```

Reply:
have changed to log & reject